src/sage/matrix/matrix_cmr_sparse.pyx

```diff
@@ -377,8 +377,62 @@ cdef class Matrix_cmr_chr_sparse(Matrix_cmr_sparse):
         sum.set_immutable()
         return sum
 
-    def two_sum(self, other, *args):
-        raise NotImplementedError
+    def two_sum(first_mat, second_mat, first_marker, second_marker):
+        r"""
```

Comment:
This "marker" terminology coming from CMR is terrible. Can we find a better name for these arguments?

Reply:
How about using col_index, row_index?

Reply:
Check how methods such as `submatrix` and `matrix_from_rows_and_columns` call their arguments

Reply:
I change it to column and row following the arguments in submatrix